rollbar/__init__.py

```diff
@@ -1606,3 +1632,32 @@ def _wsgi_extract_user_ip(environ):
     if real_ip:
         return real_ip
     return environ['REMOTE_ADDR']
+
+
+_feature_flags = []
```

Comment:
We should still make this threaded safe at some moment

Reply:
done!